game-app/game-core/src/test/java/games/strategy/triplea/xml/TestMapGameData.java

```diff
@@ -27,6 +27,7 @@ public enum TestMapGameData {
   REVISED("revised_test.xml"),
 
   VICTORY_TEST("victory_test.xml"),
+  VICTORY_TEST_SHOULD_SAVE_UP_FOR_A_FLEET("victory_test_shouldSaveUpForAFleet.xml"),
```

Comment:
created a contrived example map by copying victory_test.xml and removing land connections to Italy territory as well as removing most seaDefenseOptions by removing all sea unit productionRules except transport for Italy. Also set cost for transport at 0 so maxShipCost would not be reassigned as required to reproduce bug.